grunt/config/watch.js

```diff
@@ -23,7 +23,14 @@ module.exports = {
             '!<%= sourcedir %>extensions/extensions.js',
             '!<%= sourcedir %>menu/menu.js',
             '!<%= sourcedir %>theme/theme.js',
-            '!<%= sourcedir %>templates/templates.js'
+            '!<%= sourcedir %>templates/templates.js',
+            '!<%= sourcedir %>core/js/scriptLoader.js',
+            '!<%= sourcedir %>core/js/libraries/require.js',
+            '!<%= sourcedir %>core/js/libraries/modernizr.js',
+            '!<%= sourcedir %>core/js/libraries/json2.js',
+            '!<%= sourcedir %>core/js/libraries/consoles.js',
+            '!<%= sourcedir %>core/js/libraries/jquery.js',
+            '!<%= sourcedir %>core/js/libraries/jquery.v2.js'
```

Comment:
We need to make sure that after making change to these specific files should not trigger a `javascript:dev` task because we are just intended to copy them directly into the build folder.

I haven't used glob here because there are few other libraries available (e.g. `backbone.js`, etc..) which needs to be watched for there changes as they becomes a part of `adapt.min.js`.